### PR TITLE
Beta docs for LSE 2

### PR DESCRIPTION
**Pull request by smoreface**
Beta docs for LSE 2.0, with
- lots of new topics
- fine-tuned improvements to the docs site layout
- new diagrams

### CODE REVIEW

File: docs/source/guide/api_migrate.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+When you invite people to join your organization, workspace (formerly team), and projects, there are many changes. 
+
+### Update tokens in use
+The old tokens are no longer supported. Make a request reset your organization token used to create invitation links. 
+
+See the [reset organization token API endpoint documentation](/api#operation/api_invite_reset-token_create).
+
+### Updated invite to organization endpoint
+
+The endpoint that you use to generate invitations to your Label Studio Enterprise organization has changed from `https://app.heartex.ai/organization/welcome/<token>` to `http://localhost:8000/api/invite`.
```
Comment (niklub) [inline docs/source/guide/api_migrate.md:130]: please check gdocs comments

File: docs/source/guide/api_migrate.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+
+## Create and update external and cloud storage
+Some endpoints have been updated and some payload parameters are different when performing actions with the storage API.
+
+### Updates to storage endpoints
+
+When you want to retrieve information about a storage configuration, specify the type of storage in the API endpoint. 
+
+Instead of `/api/storages/<int:pk>/`, call `api/storages/s3/` for Amazon S3 storage connections across the organization.
```
Comment (niklub) [inline docs/source/guide/api_migrate.md:149]: int:pk -> storage_id 
api/storages/s3/ -> api/storages/s3/<storage_id>

File: docs/source/guide/api_migrate.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+| Previous parameter | New parameter |
+| --- | --- |
+| path | bucket |
+| regex | regex_filter |
+| data_key | None. Instead, BLOBs are attached to the first available object tag | 
+
+
+## Create projects
+
+With the change from teams to workspaces, the `Team_id` parameter is no longer supported in the POST payload to create a project using the API.
```
Comment (niklub) [inline docs/source/guide/api_migrate.md:174]: team_id lowercase

File: docs/source/guide/api_migrate.md [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+- `aggregated_ids` is deprecated
+- `ground_truth` is deprecated
+- `result` is no longer a double list `"result": [[... ]]` and is now a single list `“result”: [...] `
+
+#### Previous version response
+
+```json
+"completions": [ 
+ {
+ “----DEPRECATED SECTION---
```
Comment (niklub) [inline docs/source/guide/api_migrate.md:67]: I misled you -this section was actually removed

File: docs/source/guide/api_migrate.md [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+
+
+## Export data
+
+The export endpoint has changed, and so have the available options for that endpoint and the response parameters.
+
+### Updated export endpoint
+To export annotations from Label Studio Enterprise, you must call a new endpoint.
+
+Requests made to `/api/projects/pk/results` fail. Instead, call `/api/projects/pk/export?exportType=JSON`. See the [export API endpoint documentation](/api#operation/api_projects_export_read).
```
Comment (niklub) [inline docs/source/guide/api_migrate.md:38]: pk -> <project_id>

File: docs/source/guide/api_migrate.md [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+
+## Import data
+
+One endpoint has been deprecated and the payload and response have small updates. 
+
+### Update bulk task import calls
+
+The endpoint `/api/projects/<int:project_id>/tasks/bulk` still works, but is deprecated and will be removed in the future.
+
+Update calls to that endpoint to use the `/api/projects/<int:pk>/import` endpoint instead. See the [import task data API endpoint documentation](/api#operation/api_projects_import_create).
```
Comment (niklub) [inline docs/source/guide/api_migrate.md:21]: <project_id>

File: docs/source/guide/api_migrate.md [hunk 7]
```diff
@@ -0,0 +1,181 @@
...
+
+### Updates to storage endpoints
+
+When you want to retrieve information about a storage configuration, specify the type of storage in the API endpoint. 
+
+Instead of `/api/storages/<int:pk>/`, call `api/storages/s3/` for Amazon S3 storage connections across the organization.
+
+See all the storage API endpoints:
+- [Get Amazon S3 storage](/api#operation/api_storages_s3_list).
+- [Get Microsoft Azure storage](/api#operation/api_storages_azure_list).
```
Comment (niklub) [inline docs/source/guide/api_migrate.md:153]: please check gdocs comments

**Review by niklub — CHANGES_REQUESTED** [review]

File: docs/source/guide/api_migrate.md (hunk 5 above)
Comment (smoreface) [reply docs/source/guide/api_migrate.md:38]: will fix throughout

File: docs/source/guide/api_migrate.md (hunk 2 above)
Comment (smoreface) [reply docs/source/guide/api_migrate.md:149]: this is to get a specific storage ID, but you can also call it without storage ID to get all storages for the organization

File: docs/source/guide/api_migrate.md [hunk 8]
```diff
@@ -0,0 +1,180 @@
+ ---
+title: <img src="/images/LSE/en.svg" width=67 height=18 alt="Enterprise" style="vertical-align:middle"/> Update scripts and API calls
+type: guide
+order: 910
+meta_title: Update scripts and API calls to new version
+meta_description: Label Studio Enterprise documentation about updates and changes to the API endpoints in version 2.0. 
+---
+ 
+ With the new version of Label Studio Enterprise, you must update your scripts and API calls to match new API endpoints and formats. Some endpoints are new, some arguments for existing endpoints are deprecated and removed, and some payloads have changed for POST requests.
```
Comment (niklub) [inline docs/source/guide/api_migrate.md:9]: Ah one important thing we need to add at the top - is that all object IDs are changed

**Review by niklub — APPROVED** [review]

**Pull request merged**